### DISCUSSION TITLE
Add private account support

### DISCUSSION THREAD

**Discussion by bravetree** [community/community#4170] [Mobile]
I hope add a function to use my own git server.

**Comment by alcere** [comment]
Hi! Thanks for your feedback! Can you elaborate, please?

**Comment by bravetree** [comment]
I install a git server on my own Ubuntu server. So I want use the github app for iOS, iPadOS and Android to connect my own git server.
获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Alfredo ***@***.***>
发送时间: 星期一, 六月 14, 2021 3:31 下午
收件人: github/feedback
抄送: Qianhe; Author
主题: Re: [github/feedback] Add private account support (#4170)


Hi! Thanks for your feedback! Can you elaborate, please?

―

**Reply by alcere** [reply]
We are sorry, this isn’t currently on our roadmap. we’ve added your notes to our internal issues
